apps/docs/pages/guides/auth/managing-user-data.mdx

```diff
@@ -38,6 +38,10 @@ Primary keys are **guaranteed not to change**. Columns, indices, constraints or
 
 You may delete users directly or via the management console at Authentication > Users. Note that deleting a user from the `auth.users` table does not automatically sign out a user. As Supabase makes use of JSON Web Tokens (JWT), a user's JWT will remain "valid" until it has expired. Should you wish to immediately revoke access for a user, do considering making use of a Row Level Security policy as described below.
 
+<Admonition type="caution">
+Please ensure that users do not own any objects before attempting to delete them. If a deletion attempt is made on an owned object, the `storage.objects` table  will raise an error as it contains a foreign key reference to the `auth.users` table.You can individually delete objects using the GUI and you can also empty all objects in a bucket by calling the `.storage.emptyBucket('<bucket_name>')` method in the JavaScript SDK.
```

Comment:
"Please ensure that users do not own any objects before attempting to delete them"

At this point I don't know what objects you're talking about.

Reply:
There are a few grammatical errors in here too (no space between `table.You` , extra spaces between `table will`)

Reply:
Here is something that I would have written

> You cannot delete a user if they are the owner of any objects in Supabase Storage.
>
> You will encounter an error when you try to delete an Auth user that owns any Storage objects. If this happens, try deleting all the object for that user, or reassign ownership to another user.

Reply:
Thanks for the feedback - I'll be mindful to do more proofreading the next time round.

I've applied the suggestion with an edit - let me know if that works.